app/assets/stylesheets/modules/site-nav-maps.css.scss

```diff
@@ -0,0 +1,12 @@
+.nav__menu-button--hamburger {
+  background: url("../../images/icons/menu-icon-white.svg") center no-repeat;
+
+  .menu-collapsed & {
+    background-image: url("../../images/icons/menu-icon.svg");
+  }
+}
+
+.nav__menu-button--filter {
+  background: url("../../images/icons/filter-icon.svg") center / auto 17px
+    no-repeat;
+}
```

Comment:
these things are in separate files loaded just for maps OR static pages because of the different image paths

Reply:
like I said I didn't bother with organizing the stylesheets for maps and static in a neater way at this point, but will when more things like this occur. although with the redesign coming there's little sense to do this for things that will change